packages/components/src/sites-table/use-sites-table-sorting.tsx

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Does it make sense to have some tests for the various types of data inputs this selector will support?

Reply:
Do you mean to validate `sort` parameter values and handle incorrect ones somehow? Or to add tests for the sorting feature?

Reply:
More the latter: tests for the sorting feature, and different values of data it can sort.

Reply:
Thanks, I've added tests for a few cases.